daml-lf/archive/da/daml_lf_1.proto

```diff
@@ -374,9 +374,11 @@ enum BuiltinFunction {
   TO_TEXT_DATE = 71;
   TO_QUOTED_TEXT_PARTY = 63; // legacy, remove in next major version
   TO_TEXT_PARTY = 94; // *Available Since version 1.2*
+  TO_TEXT_CODE_POINTS = 105;  // *Available Since version 1.dev*
```

Comment:
Will anybody ever update this? Should we just say it'll be in 1.6?

Reply:
I will take care of that when I bump 1.6 for enum types.